data/card_catalogue/unique-1.json

```diff
@@ -304,27 +381,23 @@
     "upgrade": "",
     "tier": "blue",
     "details": {
-      "rules": "Summon a pigeon to perform one simple task",
-      "flavour": "Will try not to be distracted by breadcrumbs.",
+      "rules": {
+        "active": "Summon a pigeon to perform a simple task",
+        "passive": ""
+      },
+      "flavour": "He will try his best not to be distracted by breadcrumbs.",
       "image": "assets/core/card/illustration/Pigeon Partner.png"
     }
   },
-  {
-    "title": "You're Toast",
-    "upgrade": "",
-    "tier": "blue",
-    "details": {
-      "rules": "Summons a slice of toast the size of a door",
-      "flavour": "Roughly 13000 calories.",
-      "image": "assets/core/card/illustration/You're Toast.png"
-    }
-  },
```

Comment:
as per 2788eabe1cd2870e39c98c02024e67296fe84d07, To be discussed in

 - #110

Also as we've done away with the pigeon game
Also as I did not like the picture I made for it